soundata/jams_utils.py

```diff
@@ -37,42 +26,10 @@ def jams_converter(
             resulting jam object will not validate.
         spectrogram_path (str or None):
             A path to the corresponding spectrum file, or None.
-        beat_data (list or None):
-            A list of tuples of (annotations.BeatData, str), where str describes
-            the annotation (e.g. 'beats_1').
-        chord_data (list or None):
-            A list of tuples of (annotations.ChordData, str), where str describes the annotation.
-        note_data (list or None):
-            A list of tuples of (annotations.NoteData, str), where str describes the annotation.
-        f0_data (list or None):
-            A list of tuples of (annotations.F0Data, str), where str describes the annotation.
-        section_data (list or None):
-            A list of tuples of (annotations.SectionData, str), where str describes the annotation.
-        multi_section_data (list or None):
-            A list of tuples. Tuples in multi_section_data should contain another
-            list of tuples, indicating annotations in the different levels
-            e.g. ([(segments0, level0), '(segments1, level1)], annotator) and a str
-            indicating the annotator
-        tempo_data (list or None):
-            A list of tuples of (float, str), where float gives the tempo in bpm
-            and str describes the annotation.
-        event_data (list or None):
-            A list of tuples of (annotations.EventData, str), where str describes the annotation.
-        key_data (list or None):
-            A list of tuples of (annotations.KeyData, str), where str describes the annotation.
-        lyrics_data (list or None):
-            A list of tuples of (annotations.LyricData, str), where str describes the annotation.
-        tags_gtzan_data (list or None):
-            A list of tuples of (str, str), where the first srt is the tag and the second
-            is a descriptor of the annotation.
-        tags_open_data (list or None):
-            A list of tuples of (str, str), where the first srt is the tag and the second
-            is a descriptor of the annotation.
-        metadata (dict or None):
-            A dictionary containing the clip metadata.
         tags (list or None):
             A list of tuples of (annotations.Tags, str), where the str describes the annotation.
-
+        events (list or None):
+            A list of tuples of (annotations.Events, str), where the str describes the annotation.
```

Comment:
`events` and `tags` are not list of tuples right now. Instead, these are annotations.Events and annotations.Tags, right @magdalenafuentes ?. Should we accept tuple to include description? Should we expect a list if you have different sets of tags or events?

Reply:
Good catch! You’re right, it should say annotations.Tags and Events. Can you update it?